Cakes-Co/index.html

```diff
@@ -1,16 +1,129 @@
 <!DOCTYPE html>
 <html lang="en">
-
-<head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
     <title>Responsive Cake webpage</title>
     <!-- Add a link to your css file here -->
-</head>
+    <link rel="stylesheet" href="style.css" />
+  </head>
 
-<body>
-    <!-- Add your markup here -->
-</body>
+  <body>
+    <!-- HEADER SECTION -->
+    <header>
+      <img
+        src="/Cakes-Co/img/cake logo.png"
+        class="cupcake-logo"
+        alt="cake-picture"
```

Comment:
Great work on incorporating alt attributes for your images! In terms of improving your alt attributes even further, I would suggest using more descriptive text instead of generic terms like "image" or "picture".
For example, instead of using `<img src="example.jpg" alt="Image">`, you could use something like `<img src="example.jpg" alt="A smiling family enjoying a picnic in the park">.`